app/controllers/admin/createEvents.py

```diff
@@ -53,12 +51,9 @@ def createEvent():
             flash(validationErrorMessage, 'warning')
             return redirect(url_for("admin.createEventPage", program=2)) #FIXME: have this redirect to main programs page (or some appropriate non admin page).
 
-
-
 def selectFutureTerms(currentTermid):
     futureTerms = (Term.select().where(Term.id >= currentTermid)
                                 .where((Term.year <= (Term.get_by_id(currentTermid)).year + 2)))
-
-    listOfTerms = [future.description for future in futureTerms]
+    listOfTerms = [future for future in futureTerms]

```

Comment:
This is redundant. If you really needed a list instead of an iterable query object you could do `listOfTerms = list(futureTerms)`, but I don't think you need even that.

Reply:
we fixed this but based on the slack conversations it looks like this will change.